packages/protocol/test/TaikoL1.sim.sol

```diff
@@ -3,7 +3,6 @@ pragma solidity ^0.8.18;
 
 import {Test} from "forge-std/Test.sol";
 import {console2} from "forge-std/console2.sol";
-import {FoundryRandom} from "foundry-random/FoundryRandom.sol";
```

Comment:
please also remove it from .gitmodules

Reply:
<img width="1049" alt="Screenshot 2023-04-26 at 13 27 44" src="https://user-images.githubusercontent.com/99078276/234477810-6b86a9fe-cc84-418a-b028-b2610af6379d.png">